src/content-script.ts

```diff
@@ -0,0 +1,39 @@
+function injectScript(path: string) {
+  const script = document.createElement("script");
+  script.setAttribute("type", "text/javascript");
+  script.setAttribute("src", path);
+  document.body.appendChild(script);
+}
+
+document.addEventListener("DOMContentLoaded", (_e) => {
+  injectScript(chrome.runtime.getURL("build/injected.js"));
+});
```

Comment:
does the injection not work if we fire right on `document_start` as opposed to `DOMContentLoaded`? I think as written, we still have a race condition with Monaco being loaded. I think the ideal would be: inject script -> register an event on DOMContentLoaded to look for Monaco -> DOMContentLoaded fires, Monaco is loaded -> onDidCreateEditor is caught by our callback.